sdk/opendp/whitenoise/ast/validate.py

```diff
@@ -53,22 +86,35 @@ def check_groupkey(self):
         if (len(gbk) > 0) and (len(gbk) == len(gc)):
             raise ValueError("GROUP BY must include more than key columns: " + ", ".join([str(g) for g in gc]))
 
+    def check_select_relations(self):
+        rel_nodes = self.query.select.find_nodes(SqlRel)
+        if (len(rel_nodes)) > 0:
+            raise ValueError("We don't support subqueries in the SELECT clause")
+
+    def check_order_relations(self):
```

Comment:
I could not find any use of either of these two functions. Are they going to be used in a follow up PR?

Reply:
Oops, I decided not to validate ORDER BY, since it's done in post-processing and has no impact on rewriter or privacy.  I've removed this check now.